cmd.js

```diff
@@ -85,6 +90,28 @@ async function cmd (argv, banner = defaultBanner) {
   args.argv = subprocessArgv
   args.pathToNodeBinary = pathToNodeBinary
 
+  if (typeof args.sourceMaps === 'undefined') {
+    args.sourceMaps = null
+  } else {
+    if (args.sourceMaps !== '') {
+      const sourceMapFile = resolve('.', args.sourceMaps)
+      if (fs.existsSync(sourceMapFile)) {
+        try {
+          args.sourceMaps = JSON.parse(fs.readFileSync(sourceMapFile))
+        } catch (err) {
+          console.error(`Couldn't parse source map file ${sourceMapFile}, is it valid json?`)
+        }
```

Comment:
I suggest to remove the try / catch. Not only could it be invalid json, it could also be a permission error and a wrong path. On top of that the program should close when some of the arguments can not be properly identified.

Reply:
Will do